app/src/main/java/com/tari/android/wallet/ui/activity/CreateWalletActivity.kt

```diff
@@ -0,0 +1,208 @@
+package com.tari.android.wallet.ui.activity
```

Comment:
Please add the license text before the package declaration in every source code file.